### ISSUE TITLE
Keyboard navigation in the data view

### ISSUE THREAD

**Issue by ikelos** [Mahlet-Inc/hobbits#136]
The data view didn't seem to respond to arrow keys or page up/page down.  This makes traversing very large data sets extremely difficult.  Adding support for keyboard navigation to the data views would make analysis of large unknown/mixed-data binaries much easier.  Something like Up/Down shifting by 256 bits at a time and page up/page down shifting by a scrollbar notch each time?

**Comment by hello-adam** [comment]
yeah, that makes sense. there could probably also be more navigation options and hotkeys in the right-click navigation menu
